src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,13 +7,13 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.17.0-preview</Version>
+		<Version>1.18.0-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
-- Added support for using sever level rule in vulnerability assessment baseline operations.
 - Added support for using sever level Threat Detection.
+- Enable customers to upload their TDE certificate to CMS
```

Comment:
Customers do not know what CMS is. This release note should be more like, "Added support for uploading TDE certificate into servers and managed instances."

Reply:
You are right, will address